mlx/xunit2rst.py

```diff
@@ -186,9 +195,9 @@ def create_parser():
                             help='Optional prefix to add to item IDs')
     arg_parser.add_argument("--trim-suffix", action='store_true',
                             help="If the suffix of the --prefix argument ends with '_-' it gets trimmed to '-'")
-    arg_parser.add_argument("--unit-or-integration", action='store',
```

Comment:
can we keep this type for backwards compatibility and map it to type, while maybe printing a warning/message to user about deprecation?

Reply:
Done in e8049fa3d1bcec89215a24c25b8c431c93a60706. Let's release these changes in version 1.0.0.